lib/src/merge.rs

```diff
@@ -345,6 +345,12 @@ impl<T> Merge<T> {
         self.values.iter_mut()
     }
 
+    /// Creates a new merge by applying `f` to each remove and add.
+    pub fn into_map<U>(self, f: impl FnMut(T) -> U) -> Merge<U> {
```

Comment:
nit: `into_map` sounds like returning a `map`. `map_owned()` or something might be better.

Reply:
Thanks! `map_owned` sounds great!